test/Microsoft.ML.Tests/Scenarios/IrisPlantClassificationTests.cs

```diff
@@ -3,33 +3,45 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Data;
-using Microsoft.ML.Legacy.Models;
-using Microsoft.ML.Legacy.Trainers;
-using Microsoft.ML.Legacy.Transforms;
+using Microsoft.ML.RunTests;
 using Xunit;
-using TextLoader = Microsoft.ML.Legacy.Data.TextLoader;
 
 namespace Microsoft.ML.Scenarios
 {
-#pragma warning disable 612, 618
     public partial class ScenariosTests
     {
         [Fact]
-        public void TrainAndPredictIrisModelTest()
+        public void New_TrainAndPredictIrisModelTest()
```

Comment:
>New_ [](start = 20, length = 4)

I believe "New" prefix was used because we had two APIs but now the old API is gone and there is just one API, do you feel we still need to prefix test with "New"?

Reply:
I added it during development when I had both tests in this file.  Forgot yo drop the 'New_' prefix after deleting the old test.  Will fix.

---
In reply to: [245111276](https://github.com/dotnet/machinelearning/pull/2008#discussion_r245111276) [](ancestors = 245111276)

Reply:
fixed

---
In reply to: [245113327](https://github.com/dotnet/machinelearning/pull/2008#discussion_r245113327) [](ancestors = 245113327,245111276)